AppCenter/AppCenter/Internals/Util/MSPerformSelectorUtil.m

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSPerformSelectorUtil.h"
+#import "MSAppCenterInternal.h"
+
+@implementation MSPerformSelectorUtil
+
++ (void)performSelectorOnMainThread:(NSObject *)source withSelector:(SEL)selector withObjects:(NSObject *)objects, ... {
```

Comment:
Should probably rename to something like `performSelectorOnMainThread:fromTarget:withObjects`

Reply:
I don't understand...
Should it be:
`performSelectorOnMainThread:(NSObject *)source fromTarget:(SEL)selector withObjects:(NSObject *)objects, ... `
or
`performSelectorOnMainThreadFromTarget:(NSObject *)source withSelector:(SEL)selector withObjects:(NSObject *)objects, ... `
?

Reply:
`+ (void)performSelectorOnMainThread:(SEL)selector fromTarget:(NSObject *)target withObjects:(NSObject *)objects, ...`

Reply:
Updated